scripts/lib/build-pgroonga

```diff
@@ -2,12 +2,12 @@
 set -x
 set -e
 
-PGROONGA_VERSION="2.2.3"
+PGROONGA_VERSION="2.2.8"
 
 cd "$(mktemp -d)"
 
 wget https://packages.groonga.org/source/pgroonga/pgroonga-"$PGROONGA_VERSION".tar.gz
-tar xvf pgroonga-"$PGROONGA_VERSION".tar.gz
+tar xf pgroonga-"$PGROONGA_VERSION".tar.gz
```

Comment:
Let's do this change as a separate commit, since it's unrelated if also useful.

(Generally, when you're writing "Also, ... " in a commit message, that's a sign you're changing two unrelated things and should be doing more commits).

Reply:
Done.